core/src/commonMain/kotlin/com/sourcepoint/mobile_core/storage/Repository.kt

```diff
@@ -56,8 +56,8 @@ class Repository(private val storage: Settings) {
         get() = storage[USPSTRING_KEY]
         set(value) { storage[USPSTRING_KEY] = value }
 
-    var state: State
-        get() = Json.decodeFromString<State?>(storage.getString(SP_STATE_KEY, defaultValue = "{}")) ?: State()
+    var state: State?
+        get() = Json.decodeFromString<State?>(storage.getString(SP_STATE_KEY, defaultValue = ""))
```

Comment:
@andresilveirah I'm almost 100% sure that it will fail to decode `""` as `State?`. Better use "{}" as a default value

Reply:
You are right, it'll fail to decode and that will, in turn, return `null` (that's why I added the `?` to its type).